dev/tests/api-functional/testsuite/Magento/GraphQl/Review/GetProductReviewsTest.php

```diff
@@ -199,6 +199,63 @@ public function testProductReviewRatings()
         self::assertNotEmpty($items[0]['reviews']['items']);
     }
 
+    /**
+     * @magentoApiDataFixture Magento/Review/_files/different_reviews_different_stores.php
+     * @magentoApiDataFixture Magento/Store/_files/second_store.php
```

Comment:
It looks like your first fixture depends on the second one. That's the main reason of the failing tests. Try to change the order of fixtures loading